roles/realm_join/tasks/realm_join_ad.yml

```diff
@@ -0,0 +1,121 @@
+---
+- name: Ensure pexpect to answer password
+  become: yes
+  package:
+    name: pexpect
```

Comment:
On EL8:
```console
[root@a8f349df7349 /]# yum install pexpect
Failed to set locale, defaulting to C.UTF-8
Last metadata expiration check: 0:00:08 ago on Tue Jan 12 12:57:58 2021.
No match for argument: pexpect
Error: Unable to find a match: pexpect
```

similar on Fedora 33 and Debian 10, where is this package coming from?

Should this be `python3-pexpect`?

Reply:
It should .... python3-pexpect on RedHat > 8 and Fedora and python-pexpect on Debian.